esports_data_from_rimble/main.go

```diff
@@ -74,42 +64,29 @@ type SecretArgs struct {
 	RimbleAPIKey string `json:"api_key"`
 }
 
-type MatchWinnerArgs struct {
```

Comment:
Neither `match-winner.json` or `team-kill-diff.json` fn call examples require inputs now that we are not passing in an expected date and matchID